packages/components/pages/f-self-exclusion/src/components/SelfExclusion.vue

```diff
@@ -231,6 +233,7 @@ export default {
             isOpenAlertSuccess: false,
             isOpenAlertError: false,
             isFormVisible: false,
+            alcoholExclusionState: '',
```

Comment:
What's the purpose of this value out of interest?

Reply:
Removed, as it was not used. Thank you for addressing.